transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/util/ImportELPackageUpdater.java

```diff
@@ -0,0 +1,216 @@
+			if (!shouldAddImportPackage(pomModel)){ //$NON-NLS-1$
+				return; 
+			}
+			managePlugins(pomModel);
+
+			try (OutputStream out = new BufferedOutputStream(new FileOutputStream(pomFile))) {
+				MavenPlugin.getMaven().writeModel(pomModel, out);
+				project.getFile(IMavenConstants.POM_FILE_NAME).refreshLocal(IResource.DEPTH_ZERO, monitor);
+			}
+		} catch (CoreException | XmlPullParserException | IOException e1) {
+			Activator.error(e1);
+			return;
+		}
+	}
+
+	boolean shouldAddImportPackage(Model pomModel) {
+		return !"war".equals(pomModel.getPackaging()) && isCamelDependencyHigherThan63(pomModel);
+	}
+
+	private boolean isCamelDependencyHigherThan63(Model pomModel) {
+		try{
+			Build build = pomModel.getBuild();
+			Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+			Plugin plugin = pluginsByName.get(ORG_APACHE_CAMEL+":"+CAMEL_CORE); //$NON-NLS-1$
+			if(plugin == null || plugin.getVersion() == null){
+				return true;
+			}
+			String pluginVersion = plugin.getVersion();
+			if(pluginVersion.startsWith("${") && pluginVersion.endsWith("}") && pomModel.getProperties() != null){
+				String camelCoreVersionFromProperty = pomModel.getProperties().getProperty(pluginVersion.substring(2, pluginVersion.length() -1));
+				if(camelCoreVersionFromProperty ==  null){
+					return true;
+				} else {
+					return isVersionHigherThanLimitForImportPackage(camelCoreVersionFromProperty);
+				}
+			} else {
+				return isVersionHigherThanLimitForImportPackage(pluginVersion);
+			}
+		} catch(IllegalArgumentException e){
+			Activator.log(new Status(IStatus.INFO, Activator.PLUGIN_ID, "The version of camel-core has not been resolved correctly.", e));
+			return true;
+		}
+	}
+
+	private boolean isVersionHigherThanLimitForImportPackage(String camelCoreVersion) {
+		return new Version(camelCoreVersion).compareTo(new Version(LIMIT_CAMEL_VERSION_FROM_WHICH_NEED_TO_ADD_IMPORT_PACKAGE)) >= 0;
+	}
+
+	private void managePlugins(Model pomModel) throws XmlPullParserException, IOException {
+		Build build = pomModel.getBuild();
+		Map<String, Plugin> pluginsByName = build.getPluginsAsMap();
+		Plugin plugin = pluginsByName.get(ORG_APACHE_FELIX+":"+MAVEN_BUNDLE_PLUGIN); //$NON-NLS-1$
+		if (plugin == null) {
+			plugin = new Plugin();
+			plugin.setGroupId(ORG_APACHE_FELIX); //$NON-NLS-1$
+			plugin.setArtifactId(MAVEN_BUNDLE_PLUGIN); //$NON-NLS-1$
+			plugin.setVersion(MAVEN_BUNDLE_PLUGIN_VERSION); //$NON-NLS-1$
+			plugin.setExtensions(true);
+			build.addPlugin(plugin);
+		}
+		manageConfigurations(plugin);
+	}
+
+	private void manageConfigurations(Plugin plugin) throws XmlPullParserException, IOException {
+		Xpp3Dom config = (Xpp3Dom)plugin.getConfiguration();
+		if (config == null) {
+			config = Xpp3DomBuilder.build(new ByteArrayInputStream((
+					"<configuration>" + //$NON-NLS-1$
+							"    <excludeDependencies>false</excludeDependencies>" + //$NON-NLS-1$
+							"    <archive>" + //$NON-NLS-1$

```

Comment:
The only reason I added it in the first place was because it was in the sample that Lars originally pointed me at.  I had no idea as to its us, so I defer to @lhein if he says it's not needed.